### DISCUSSION TITLE
Generic Device Tracker

### DISCUSSION THREAD

**Discussion by miguelangellv** [frenck/spook#396] [Ideas]
I am looking to track my car position and I see that spook allow me update zones, but, I think that a equivalente device tracker could be logical, because a car is not a "zone", is a "device", and, when the car is en home, is "invisible" in map.

**Comment by jsnfwlr** [comment]
you would need a device of some sort in the car, and that device would report it's location to HA. This probably isn't something Spook should be adding support for
